aries_cloudagent/core/tests/test_dispatcher.py

```diff
@@ -1,7 +1,7 @@
 import json
 
 from async_case import IsolatedAsyncioTestCase
-import mock as async_mock
+from asynctest import mock as async_mock
```

Comment:
This shouldn't be necessary, maybe you need to install requirements.dev.txt?

Reply:
Updated to use asyncMock().